python/ray/_raylet.pyx

```diff
@@ -718,12 +718,7 @@ cdef int64_t restore_spilled_objects_handler(
             exception_str = (
                 "An unexpected internal error occurred while the IO worker "
                 "was restoring spilled objects.")
-            logger.exception(exception_str)
-            ray._private.utils.push_error_to_driver(
-                ray.worker.global_worker,
-                "restore_spilled_objects_error",
-                traceback.format_exc() + exception_str,
-                job_id=None)
+            logger.exception(traceback.format_exc() + exception_str)
```

Comment:
You don't need to include the traceback with logger.exception, it's automatically included.